Resources/Prototypes/_Ronstation/Procedural/salvage_loot.yml

```
@@ -0,0 +1,45 @@
+# Seed loot table
+- type: salvageLoot
+  id: SalvageLoot
```

Needs a rename to something like "SalvageLootRonstation".
```suggestion
  id: SalvageLootRonstation
```